core/templates/pages/preferences-page/modal-templates/edit-profile-picture-modal.controller.ts

```diff
@@ -73,8 +73,7 @@ angular.module('oppia').controller('EditProfilePictureModalController', [
     };
 
     $scope.onInvalidImageLoaded = function() {
-      $scope.uploadedImage = null;
```

Comment:
why remove this?

Reply:
Because `$scope.reset` is already doing the same steps